scripts/generate_syncset.py

```diff
@@ -92,15 +94,17 @@ def process_yamls(name, directory, obj, platform="", sss_mode="sync"):
             dirpaths.append(dirpath)
 
     for dirpath in sorted(dirpaths):
-        sss_mode = "sync"
-        if "UPSERT/" in dirpath:
-            sss_mode = "upsert"
-
-        platform = ""
-        if "/gcp" in dirpath:
-            platform = "gcp"
-        if "/aws" in dirpath:
-            platform = "aws"
+        # load sss_config if it exists
+        sss_config = {}
+        path_sss_config = os.path.join(dirpath, sss_config_filename)
+        if os.path.exists(path_sss_config):
+            sss_config = get_yaml(path_sss_config)
+
+        # initialize defaults for sss_config
+        if "matchLabels" not in sss_config:
+            sss_config["matchLabels"] = {}
```

Comment:
Shouldn't we default to the `managed = true` label in case none is specified in `sss-config.yaml`?

Otherwise, we'll apply the SSS in non-managed clusters as well

Reply:
Defaults are in the SSS template.  https://github.com/openshift/managed-cluster-config/blob/master/scripts/templates/selectorsyncset.yaml#L10-L13

Reply:
hmm, nice